redbot/cogs/mod/kickban.py

```diff
@@ -369,19 +379,34 @@ def remove_processed(ids):
             user = discord.Object(id=user_id)
             audit_reason = get_audit_reason(author, reason)
             queue_entry = (guild.id, user_id)
-            try:
-                await guild.ban(user, reason=audit_reason, delete_message_days=days)
-                log.info("{}({}) hackbanned {}".format(author.name, author.id, user_id))
-            except discord.NotFound:
-                errors[user_id] = _("User {user_id} does not exist.").format(user_id=user_id)
-                continue
-            except discord.Forbidden:
-                errors[user_id] = _("Could not ban {user_id}: missing permissions.").format(
-                    user_id=user_id
-                )
-                continue
-            else:
-                banned.append(user_id)
+            async with self.config.guild(guild).current_tempbans() as tempbans:
+                if user_id in tempbans:
+                    tempbans.remove(user_id)
+                    errors[user_id] = _("Upgraded tempban for {user_id} to a permaban.").format(
+                        user_id=user_id
+                    )
```

Comment:
This shouldn't show under errors - this is just an additional information about the ban, it's not an error. I don't have any specific way of showing it in mind though (if we *do* want to show this information).

Reply:
I think it's important to show that an upgrade happen and made some changes. Let me know what you think of them.